DEVELOPMENT.md

```diff
@@ -32,6 +32,13 @@ You must install these tools:
 1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
    managing development environments.
 
+If you're working on and changing `proto` files:
+
+1. [`protoc`](https://github.com/protocolbuffers/protobuf): For compiling
+   protocol buffers.
+1. [`protoc-gen-gogofaster`](https://github.com/gogo/protobuf/#more-speed-and-more-generated-code):
+   For generating efficient golang code out of protocol buffers.
+
```

Comment:
Should the section calling out `update-codegen` mention the flag?

Reply:
Added!